cypress/.eslintrc.json

```diff
@@ -10,6 +10,8 @@
   ],
   "rules" : {
     "import/no-extraneous-dependencies": ["error", {"devDependencies": true}],
+    "jest/expect-expect": "off",
+    "jest/valid-expect-in-promise": "off",
```

Comment:
This turns off the complaints about `it` (that there are not `expect` statements inside) and about returning promises.
Both are caused by Jest.

In my understanding, Cypress is using Chai under the hood and Jest is not needed nor used in our Cypress tests.
So we can turn the Jest rules off in the `cypress` folder (right?).
I couldn't find a one-line way though.
@safydy-r, @object-required

Reply:
no concern about this for now, thanks